project/src/main/java/com/google/sps/Models/Database.java

```diff
@@ -156,10 +160,40 @@ public static Document getDocumentByHash(String hash) {
       return null;
     }
 
-    ArrayList<Long> userIDs = getListProperty(docEntity, "userIDs");
+    Document doc = getDocumentByHash(hash);
+    ArrayList<Long> userIDs = doc.getUserIDs();
+    //ArrayList<Long> userIDs = getListProperty(docEntity, "userIDs");
     return userIDs;
   }
 
+  public static ArrayList<Long> getDocumentEditors(String hash) {
```

Comment:
There is a function in Document class itself , but this is needed here because otherwise there wont be a way to update the editors and viewers in the database itself. This is just modularizing the share feature